char repairCode
---
bool isSuccess
string detail
int32 countRepairedDetails
